libsrc/c128/isfast.s

```diff
@@ -0,0 +1,20 @@
+;
+; Marco van den Heuvel, 2018-03-19
+;
+; unsigned char isfast (void);
+; /* Returns 1 if the CPU is in 2MHz mode. */
+;
+
+        .export         _isfast
+
+        .include        "c128.inc"
+
+
+.proc   _isfast
+
+        lda     VIC_CLK_128
+        and     #$01
+        rts
```

Comment:
You need to set X, see https://github.com/cc65/wiki/wiki/Parameter-passing-and-calling-conventions#Overview

Reply:
Ah, I see, ok, will fix in a few.